.github/workflows/release.yml

```diff
@@ -124,6 +124,12 @@ jobs:
           echo "PG_DATABASE_URL=postgres://postgres:root@localhost/" >> $GITHUB_ENV
           echo "PG_EXAMPLE_DATABASE_URL=postgres://postgres:root@localhost/diesel_example" >> $GITHUB_ENV
 
+      - name: Install jq (MacOS arm64)
+        if: ${{ matrix.os == 'macos-arm64-self-hosted' && env.s3_archive_exist == '' }}
```

Comment:
so this only runs if the archive does not exist, right?

Reply:
yes